backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/ColumnTypeRdfMapper.java

```diff
@@ -131,7 +131,7 @@ Set<Value> retrieveValues(String baseURL, Row row, Column column) {
     STRING(CoreDatatype.XSD.STRING) {
       @Override
       Set<Value> retrieveValues(String baseURL, Row row, Column column) {
-        return basicRetrievalString(row.getStringArray(column.getName()), Values::literal);
+        return basicRetrievalString(row.getStringArrayPreserveDelimiter(column), Values::literal);
```

Comment:
huh? why would you prefer this over the array? And does getString() not get you what you want?

Reply:
different return type? (String instead of String[] ?) but this is so trivial, please refactor if needed, at least it works like this